Cabal/Distribution/Simple/PreProcess/Unlit.hs

```diff
@@ -35,10 +36,10 @@ classify ('>':s) = BirdTrack s
 classify ('#':s) = case tokens s of
                      (line:file:_) | all isDigit line
```

Comment:
```suggestion
                     (line:file@('"':_:_)_) | all isDigit line
```

fwiw, if we had the pattern match above, we could omit the inefficient `length`-check as well as the 'safeHead` call